README.md

```diff
@@ -57,19 +91,27 @@ It is highly recommended that you use a tool like [`nvm`](https://github.com/nvm
     $ npm run prepare
     $ npm start
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and opens up a browser window. Most changes are 
+reflected live without having to restart the server.
 
 ### Build
 
     $ npm run build
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+This command generates static content into the `build` directory and can be served using any static 
+contents hosting service.
 
 ### Adding new words to the dictionary
 
-This repository includes a _linter_, which you can think of as a spell-check that also checks code formatting and standards, and a lot more. It's possible that you will use a word in your content that is not known to the linter, and your build, or commit, will fail.
+This repository includes a _linter_, which you can think of as a spell-check that also checks code 
+formatting and standards, and a lot more. It's possible that you will use a word in your content 
+that is not known to the linter, and your build, or commit, will fail.
 
-If this happens, take a look at `project-words.txt` in the root directory of your project; if the word that the linter caught is correctly spelled, and you wish it to pass the linter's test, add it to `project-words.txt`, save, add and commit those changes, and see if it passes.
+If this happens, take a look at `project-words.txt` in the root directory of your project; if the 
+word that the linter caught is correctly spelled, and you wish it to pass the linter's test, 
+add it to `project-words.txt`, save, add and commit those changes, and see if it passes.
+
+For tidiness, please ensure you adhere to the alphabetical order established in `project-words.txt`.
 
 ### Local development
```

Comment:
Duplicate content? This appears to be the same as the "Local Development" section above.